### PR TITLE
Fix: Adjust the isetting as there are times when it works and times when it does not

### PR DESCRIPTION
**Pull request by carin91**
The appContext get lost when refreshing the page, therefore i create a localStorage with the client info.

This branch also addresses the set704

### CODE REVIEW

**Review by LinconOssa — COMMENTED** [review]
Based on what was discussed in the meeting, it is returned to make the adjustment.

**Review by LinconOssa — APPROVED** [review]
meet the requirements

File: package.json [hunk 1]
```diff
@@ -22,6 +22,7 @@
     "@testing-library/user-event": "^13.5.0",
     "chromatic": "^6.17.1",
     "formik": "^2.4.1",
+    "js-cookie": "^3.0.5",
```
Comment (cmarin001) [inline package.json:25]: we should avoid the use of this library since is not necessary at this time

**Review by LinconOssa — APPROVED** [review]
meet the requirements

**Pull request merged**